macos/settings/data_loss_prevention/schema.json

```diff
@@ -0,0 +1,434 @@
+{
+    "__feedback": "jmanifest@microsoft.com",
+    "__version": "101.13.75",
```

Comment:
That's the "from" version where this schema is supported. I try to use the _rough_ versions of wdav.pkg, starting with the one this schema is supported.

Reply:
Updated to current release: 2108-2

Reply:
Updated to the current release: 2108-2